tests/test_parser.py

```diff
@@ -7,21 +7,24 @@
 from gdtoolkit.parser import parse
 
 
-OK_DATA_DIR = "valid-gd-scripts"
-NOK_DATA_DIR = "invalid-gd-scripts"
+OK_DATA_DIR = "./valid-gd-scripts"
```

Comment:
I added the leading ./ just so it's explicit that these are relative paths

Reply:
Is it portable to e.g. Windows?